### ISSUE TITLE
path in javascript 

### ISSUE THREAD

**Issue by serman** [studioijeoma/ijeomamotion#7]
Do you have the javascript port of the path class?
is it available anywhere?

Thanks!

**Comment by studioijeoma** [comment]
Yes, I have but I'm still fixing some issues. May take another 2-3 weeks. How's everything else?

**Comment by studioijeoma** [comment]
Also the javascript library can be found at https://github.com/ekeneijeoma/ijeoma.js. please post issues there.
